### PR TITLE
fix(dashboard-customize): apply widgets to customize layout

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
스크린샷 2022-12-26 오후 12 56 53

### CODE REVIEW

**Review by song2kim — APPROVED** [review]
LGTM :)

File: src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue [hunk 1]
```diff
@@ -20,48 +20,91 @@
...
 const state = reactive({
     labelList: [] as Array<string>,
     refreshInterval: undefined,
     dateRange: {
         start: dayjs.utc().format('YYYY-MM-01'),
         end: dayjs.utc().format('YYYY-MM-DD'),
     } as DateRange,
+    isProjectDashboard: computed<boolean>(() => Boolean(props.dashboardId.startsWith('project'))),
+    dashboardInfo: {} as DashboardModel,
+    dashboardWidgetLayouts: computed<DashboardLayoutWidgetInfo[][]>(() => state.dashboardInfo?.layouts ?? []),
```
Comment (piggggggggy) [inline src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue:74]: Just question: What is the difference between using `startsWith` alone and using `Boolean()`?

**Review by piggggggggy — APPROVED** [review]
LGTM!!

File: src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue (hunk 1 above)
Comment (yuda110) [reply src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue:74]: Oh I just copied @minseolee 's code 😅
I think there's no difference.

Comment (minseolee) [reply src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue:74]: Oops, my mistake 
I'll apply it!

**Pull request merged**